packages/app/src/utils/generateAltText.ts

```diff
@@ -0,0 +1,10 @@
+import { UpsetConfig, getRows } from "@visdesignlab/upset2-core";
+import { getAltTextConfig } from "@visdesignlab/upset2-react";
+import { api } from "../atoms/authAtoms";
+
+export const generateAltText = async (state: UpsetConfig, data: any, verbosity: string, explain: string): Promise<string> => {
```

Comment:
I generally prefer to have do function declarations like this:

```suggestion
export async function generateAltText(state: UpsetConfig, data: any, verbosity: string, explain: string): Promise<string> {
```

Reply:
Also, I'm not seeing the benefit of having this in its own file. It's led to some weird naming issues:

- Here it is generateAltText
- In Body there is a new genAltText
- That gets passed into AltTextSidebar as props.generateAltText
- You then make a function called genAltText from that

We really should clean that up, because you'll hate your life if you ever need to change this lol